nautobot/dcim/templates/dcim/powerfeed_retrieve.html

```diff
@@ -41,7 +47,12 @@
                         {% if object.connected_endpoint %}
                             {{ object.connected_endpoint.parent|hyperlinked_object }}
                             ({{ object.connected_endpoint }})
-                        {% else %}
+                        {% endif %}
+                        {% if object.destination_panel %}
+                            {% if object.connected_endpoint %}<br/>{% endif %}
+                            {{ object.destination_panel|hyperlinked_object }}
+                        {% endif %}
```

Comment:
Feels like this should be a separate row in the table rather than overloading the `Connected Device` row?